### DISCUSSION TITLE
Round all tracking to a specific timeframe

### DISCUSSION THREAD

**Discussion by b12f** [solidtime-io/solidtime#233] [Feature requests]
Hello and thanks for the great tool!

Right now, when tracking time, the start and stop seconds are saved in the timestamp, but in the UI only minutes are shown. This causes discrepancies between the time shown in reporting and the cost shown. Example with an hourly wage of 100€:

![20241111_14h24m18s_grim](https://github.com/user-attachments/assets/1dbfc3c7-229c-484d-8df4-8df239c226d0)

We'd like to see a config option that automatically rounds all tracking values to at least the nearest minute, but even better would be if it could also be rounded to e.g. the next quarter hour.

**Comment by Onatcer** [comment]
We decided to move feature requests into github discussions, to separate feature requests from bugs and allow for better prioritization. If this feature is important to you, you can upvote it to let us know.

**Comment by korridor** [comment]
The development of this feature is in PR #481

**Comment by aaronczichon** [comment]
Would also love to see this feature. Nice would be, if I'm able to configure, as mentioned, to round it on a detail report for nearest hour, half hour or quarter hour.

**Comment by Onatcer** [comment]
This was released with v0.9.0, you can now apply rounding in the Reporting Overview and Reporting Detailed Views. 

https://github.com/solidtime-io/solidtime/releases/tag/v0.9.0
